http/task_service.go

```diff
@@ -511,6 +511,14 @@ func decodeGetTasksRequest(ctx context.Context, r *http.Request, orgs influxdb.O
 		req.filter.Limit = influxdb.TaskDefaultPageSize
 	}
 
+	if status := qp.Get("active"); status == "true" {
+		active := false
+		req.filter.Active = &active
+	} else if status := qp.Get("active"); status == "false" {
+		active := false
+		req.filter.Active = &active
+	}
+
```

Comment:
You will need to amend the `swagger.yml` to include this new query argument.

Reply:
good catch!